Applications/ConsoleReferenceClient/Program.cs

```diff
@@ -356,17 +358,53 @@ public static async Task Main(string[] args)
                             }
                             else
                             {
+                                bool enableDurableSubscriptions = false;
+                                int quitTimeout = 30_000;
+
+#if Enable_Durable_Subscriptions
```

Comment:
I would prefer a command line parameter over the Compiler statement

Reply:
Updated as requested.